tmp/cjs/react-jsx-dev-runtime.development.js

```diff
@@ -14,7 +14,7 @@ if (process.env.NODE_ENV !== "production") {
 'use strict';
 
 var React = require('react');
-var _assign = require('object-assign');
```

Comment:
I noticed this wasn't actually used in this file. It won't exist in 0.14 but I'm removing now since I needed to edit this part anyway.